### ISSUE TITLE
CTRL + A not working when CAPS are on

### ISSUE THREAD

**Issue by alialbaali** [JetBrains/compose-jb#1457]
Reproduce:
```kotlin
fun main() = singleWindowApplication {
    val (text, setText) = remember { mutableStateOf("") }
    TextField(text, setText)
}
```

**Comment by igordmn** [comment]
Which version of Compose do you you use, and on what OS you are running?

I can't reproduce this on Windows 11, 1.0.0-rc3

**Comment by alialbaali** [comment]
Sorry, I forgot to mention. 
OS : Linux PopOS 21.04
Compose: 1.0.0-beta5
Kotlin : 1.5.31

**Comment by igordmn** [comment]
Is CapsLock changing the keyboard layout?

I checked, and it appears that hotkeys don't work if the layout isn't English

**Comment by igordmn** [comment]
With Caps On I have reproduced it too - hotkeys don't work. Investigating...

**Comment by alialbaali** [comment]
In my case, they don't work with English nor Arabic.

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-rc10

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5164">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
